### DISCUSSION TITLE
[COMPOSITE] Ressource Planning Hebdo

### DISCUSSION THREAD

**Discussion by manufitoussiwit** [witsa/synapps#471] [Show and tell]
Un exemple de composite pour la ressource *planning hebdo*.

![hebdo](https://user-images.githubusercontent.com/35595723/165552861-91153816-54ad-4f66-ac17-2de65f2b09bb.gif)

Il est disponible dans les composites remarquables <a href="https://witsa.github.io/synapps/concepts/composite.html#planning-hebdomadaire" target="new">➡️ ici</a>

A copier dans vos composite.

Mise à jour : 
![hebdo](https://user-images.githubusercontent.com/35595723/165721401-dbdea724-5f17-4f9c-b65a-73b6b07b21bd.gif)

### Edition du 24/07/2025
- adaptation aux navigateurs moins récents. 

### Edition du 06/06/2025
- correction de l'indicateur de quart d'heure courant pour pour des fuseaux horaires différents de Paris sur les REDY.

### Edition du 03/10/2024
- correction de la taille du bouton switch auto/forcé
- correction de l'alignement vertical du bouton switch auto/forcé
- correction d'un pb qui apparaissait sur réseau lent et qui produisait des erreurs dans la transmission des périodes à la ressource.

### Edition du 09/09/2024
- correction de l'indicateur du quart d'heure en cours pour tenir compte de l'heure d'été/hiver. La précédente correction n'était pas complètement aboutie. Entre 00:00 et 2:00 du matin, l'affichage était incorrect (#759)

### Edition du 27/02/2023:
- correction de l'indicateur du quart d'heure en cours pour tenir compte heure d'été/hiver actif ou pas. Auparavant, lorsque la gestion été/hiver était désactivé, l'heure se retrouvait en UTC.

**Comment by WITNicolas** [comment]
C'est paaaarfait ça !!

**Comment by JYK-WIT** [comment]
Nickel, il tombe bien ce composite
Merci

**Comment by manufitoussiwit** [comment]
J'ai fait une mise à jour du composite!
![hebdo](https://user-images.githubusercontent.com/35595723/165721548-5017e02e-cdad-4200-bf09-d35e96f05cf1.gif)

**Reply by Mathis554** [reply]
Bonjour,
Savez vous-ci, il est possible de rajouter ces paramètres qui étaient dans les plannings de Wit ?
![image](https://github.com/witsa/synapps/assets/174346872/ef5157f2-f4e3-4e75-9c1a-1169f73be9e2)

**Comment by JYK-WIT** [comment]
A quand le déplacement type échec ?

Envoyé à partir de Courrier<https://go.microsoft.com/fwlink/?LinkId=550986> pour Windows

________________________________
De : Emmanuel FITOUSSI ***@***.***>
Envoyé : Thursday, April 28, 2022 11:23:57 AM
À : witsa/synapps ***@***.***>
Cc : Jean-Yves KNECHT ***@***.***>; Comment ***@***.***>
Objet : Re: [witsa/synapps] Composite Ressource Planning Hebdo (Discussion #471)


J'ai fait une mise à jour du composite!
[hebdo]<https://user-images.githubusercontent.com/35595723/165721548-5017e02e-cdad-4200-bf09-d35e96f05cf1.gif>

**Comment by WITNicolas** [comment]
Est ce qu'il serait possible d'avoir un composite pour le Planning par Plage Hebdomadaire ?
Ce planning est trés utilisé par les exploitants.

**Comment by manufitoussiwit** [comment]
J'ai fait une petite mise à jour pour que rien ne s'affiche tant qu'on ne choisi pas une ressource

**Reply by WITNicolas** [reply]
Est ce que tu as copié le nouveau code dans la doc en ligne, les Composites Remarquables ?

**Reply by manufitoussiwit** [reply]
ouip

**Comment by manufitoussiwit** [comment]
Mise à jour.
*Faudra prendre cette version lorsque vous utiliserez la prochaine version de Studio (1.1.5).*

**Comment by FlorentAdell** [comment]
Super Composite ! Il est super !

**Comment by FlorentAdell** [comment]
@manufitoussiwit, de mon côté il demeure encore un problème d'affichage, le planning se base sur l'heure UTC et non l'heure interne de l'appareil (cf.photos)

![image](https://user-images.githubusercontent.com/46195964/221510633-fd0e4746-ac8c-4538-9d42-1bed40fb6164.png)
![image](https://user-images.githubusercontent.com/46195964/221510699-1254ed6c-7fee-4363-9c6b-44bb0e31262f.png)

**Comment by Raphwit** [comment]
Bonjour,

Chez moi le planning s'adapte bien à l'heure de l'appareil, as-tu pensé à bien recharger la page une fois la modification du fuseau horaire effectuée ? :)

![image](https://user-images.githubusercontent.com/93767525/221529526-b5015671-ad90-4f56-aa90-e1da8fa0f82a.png)

![image](https://user-images.githubusercontent.com/93767525/221529642-fb6e8db4-53ef-4835-8db4-fffd44bebed0.png)

**Reply by FlorentAdell** [reply]
Rebonjour, oui en effet ...
![image](https://user-images.githubusercontent.com/46195964/221541223-596d032f-0ddc-4472-bd2d-fcd33719ff92.png)
![image](https://user-images.githubusercontent.com/46195964/221541335-f057220e-2573-4d9d-a853-76e72a88bb36.png)

Voici pour exemple entre mon wit et le synapps
Je vais essayer de chercher pourquoi

**Reply by Raphwit** [reply]
Effectivement, pour l'instant il est préférable de réactiver la mise à jour de l'heure automatique ici :

![image](https://user-images.githubusercontent.com/93767525/221548321-182c6a6a-fda1-43c4-a48d-2b0e64a07d22.png)


le problème a été remonté et une correction sera fournie :)

**Reply by manufitoussiwit** [reply]
Salut! 
Correction faite dans le message d'origine. 
Maintenant, si la gestion Eté hiver est décoché, l'heure en cours reste bonne.
++

**Comment by manufitoussiwit** [comment]
Une correction du composite a été faite pour prendre en compte correctement l'heure d'été/hivers entre minuit et 2h. #759

**Comment by manufitoussiwit** [comment]
### Edition du 03/10/2024
- correction de la taille du bouton switch auto/forcé
- correction de l'alignement vertical du bouton switch auto/forcé
- correction d'un pb qui apparaissait sur réseau lent et qui produisait des erreurs dans la transmission des périodes à la ressource issue : #784

**Comment by Raphael-zoreille** [comment]
Bonjour, 
serait-il possible d'ajouter le choix du retour en auto ? 
![image](https://github.com/user-attachments/assets/398c808b-bdfa-40d7-aefb-c1277d0ad39d)

Merci 😎

**Reply by manufitoussiwit** [reply]
Bien vu, on pourrait même rendre son affichage optionnel

**Comment by manufitoussiwit** [comment]
### Edition du 06/06/2025
- correction de l'indicateur de quart d'heure courant pour des fuseaux horaires différents de Paris sur les REDY.

**Reply by Raphael-zoreille** [reply]
Hello, ok parfait le problème n'y est plus. Merci

**Comment by manufitoussiwit** [comment]
### Edition du 24/07/2025
- adaptation aux navigateurs moins récents. 

En relation avec #892
